__tests__/app.test.js

```diff
@@ -52,11 +53,80 @@ describe('app routes', () => {
       ];
 
       const data = await fakeRequest(app)
-        .get('/animals')
-        .expect('Content-Type', /json/)
-        .expect(200);
+        .get('/desserts')
+        .expect('Content-Type', /json/);
+      // .expect(200);
 
       expect(data.body).toEqual(expectation);
     });
+
+    test('POST /desserts creates a new dessert', async () => {
+      const newDessert = {
+        name: 'Chocolate Chip',
+        icing: false,
+        type: 'cookie'
+      };
+
+      const data = await fakeRequest(app)
+        .post('/desserts')
+        .send(newDessert)
+        // .expect(200)
+        .expect('Content-Type', /json/);
+
+      expect(data.body.name).toEqual(newDessert.name);
+      expect(data.body.id).toBeGreaterThan(0);
+
+    });
+    test('PUT /desserts creates an updated dessert', async () => {
+      const updatedDessert = {
+        name: 'Chocolate Chip and walnut',
+        icing: false,
+        type: 'cookie'
+      };
+
+      const data = await fakeRequest(app)
+        .put('/desserts')
+        .send(updatedDessert)
+        // .expect(200)
+        .expect('Content-Type', /json/);
+
+      expect(data.body.name).toEqual(updatedDessert.name);
+      expect(data.body.id).toBeGreaterThan(0);
+    });
+    test('POST /desserts creates an updated dessert', async () => {
+      const newDessertInArray = {
+        name: 'Chocolate Chip',
+        icing: false,
+        type: 'cookie'
+      };
+
+      const data = await fakeRequest(app)
+        .put('/desserts')
+        .send(newDessertInArray)
```

Comment:
Looks like this is your POST test so the method you're calling on your API should be a `.post` instead of a `.put`